torch/ao/quantization/experimental/quantizer.py

```diff
@@ -18,6 +18,7 @@ def __init__(
         self,
         b,
         k,
+        min_val,
```

Comment:
the quantizer should take quantization parameters as arguments, e.g. https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/quantized/Quantizer.cpp#L70-L73

Reply:
@dzdang that makes sense to me, i'll do that!

Reply:
it doesn't seem this was addressed in the latest version

Reply:
@dzdang my quantizer `init` function is only taking in observer now -- is that what you meant above?

Reply:
@dzdang i interpreted the above as just passing in observer to quantizer, but i see you mean that externally observer should compute quantization parameters and then pass those into constructor for quantizer. will fix!